### ISSUE TITLE
Not all the command line options are exposed through the plugin.

### ISSUE THREAD

**Issue by dilipbiswal** [ihji/sbt-antlr4#20]
Here are the lists of command line options.
```
$ antlr4
ANTLR Parser Generator Version 4.0b3
-o <arg>                specify output directory where all output is generated
-lib <arg>              specify location of grammars, tokens files
-atn                    generate rule augmented transition network diagrams
-encoding <arg>         specify grammar file encoding; e.g., euc-jp
-message-format <arg>   specify output style for messages in antlr, gnu, vs2005
-listener               generate parse tree listener (default)
-no-listener            don't generate parse tree listener
-visitor                generate parse tree visitor
-no-visitor             don't generate parse tree visitor (default)
-package <arg>          specify a package/namespace for the generated code
-depend                 generate file dependencies
-D<option>=value        set/override a grammar-level option
-Werror                 treat warnings as errors
-Xsave-lexer            save temp lexer file created for combined grammars
-XdbgST                 launch StringTemplate visualizer on generated code
-Xforce-atn             use the ATN simulator for all predictions
-Xlog                   dump lots of logging info to antlr-timestamp.log
-XdbgSTWait             wait for STViz to close before continuing
```
I want to be able to turn warnings to errors while generating parser. But that option is not exposed through the plugin ? The maven plugin for antlr4 allows us to do that.

cc @ihji

**Comment by ihji** [comment]
@dilipbiswal Thanks for opening the issue. I expect I can work on it next week. I'm also more than happy to merge PR if it's urgent.

**Comment by dilipbiswal** [comment]
@ihji Thanks for your response. I have tried to create two prs. One for sbt-1.0 and other for sbt-013.

**Comment by ihji** [comment]
published 0.8.2 (sbt-1.0) and 0.7.13 (sbt-0.13). Thanks!
